dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -114,6 +114,10 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
         />
       </div>
     </div>
+  ) : (
+    <div className="grid h-[400px] place-items-center rounded-md bg-slate-100 dark:bg-slate-800">
```

Comment:
lets avoid using random colors, if necessary, add some of them to the tailwind.config file, but let's going to use only colors that are there